server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -506,7 +506,7 @@ export default class ServiceProviderReferralsController {
         formError = data.error
         userInputData = req.body
       } else {
-        const updatedAppointment = await this.interventionsService.recordAppointmentAttendance(
+        const updatedAppointment = await this.interventionsService.recordActionPlanAppointmentAttendance(
```

Comment:
I feel like a while back we used the term `sessionDeliveryAppointment` to refer to these sorts of appointments see Miro notes here: https://miro.com/app/board/o9J_klQS0Ds=/?moveToWidget=3074457359305932502&cot=14 (although I can also see ActionPlanSession there but I think that was the previous incarnation) - this might be outdated information though, so it's worth checking what terminology they use on the backend.

Reply:
Ah, that's good to know, thanks! It [looks like](https://github.com/ministryofjustice/hmpps-interventions-service/pull/325/files#diff-fc01173b65c46c954537c8d5a7f53411f59e0cdcb66618348cecc3fd526cd904R13) we've gone back to calling them "action plan session" again, though…